SU-SWS/stanford_paragraph_types
# Hsdo 954

# READY FOR REVIEW

# Summary
- Section Divider paragraph type styling tweaks

# Needed By (Date)
- August 2, 2017

# Urgency
- low

# Steps to Test

1. checkout branch HSDO-954
2. Add Section Divider paragraph type to Story Page content type. Padding should be 50px 0.

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
